ansible/scripts/cluster.sh

```diff
@@ -14,6 +14,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-inventory=${INVENTORY:-inventory}
+inventory=${INVENTORY:-../inventory}
 
-ansible-playbook -i ${inventory} cluster.yml $@
+ansible-playbook -i ${inventory} ../playbooks/cluster.yml $@
```

Comment:
Taking my argument back. Reading more about variables, inventory directory seems the most appropriate for ``group_vars``. As long as we avoid ``group_vars`` inside playbook directory, we are then fine.